### PR TITLE
Added Status check for mcg-ms-console pod

### PR DESCRIPTION
**Pull request by achouhan09**
Signed-off-by: Aayush Chouhan <achouhan@redhat.com>

### CODE REVIEW

**Review by naveenpaul1 — APPROVED** [review]
lgtm

**Comment by naveenpaul1** [comment]
@achouhan09 please fix lint error

File: controllers/managedmcg_controller.go [hunk 1]
```diff
@@ -697,6 +704,31 @@ func (r *ManagedMCGReconciler) updateComponentStatus() {
...
+		} else {
+			consoleStatus.State = mcgv1alpha1.ComponentPending
+		}
+	case errors.IsNotFound(err):
+		consoleStatus.State = mcgv1alpha1.ComponentNotFound
+	default:
+		r.Log.Info("error getting Console, setting component status to unknown")
+		consoleStatus.State = mcgv1alpha1.ComponentUnknown
+	}
 }
```
Comment (rexagod) [inline controllers/managedmcg_controller.go:732]: I'd suggest breaking off the Noobaa, Alertmanager, Prometheus, and Console status updation parts into their own functions, and calling them here to reduce the cognitive complexity: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/red-hat-storage_mcg-osd-deployer/76/pull-ci-red-hat-storage-mcg-osd-deployer-main-lint/1556936379416449024#1:build-log.txt%3A24.

**Review by rexagod — CHANGES_REQUESTED** [review]

**Review by rexagod — CHANGES_REQUESTED** [review]
- ~~Please replace https://github.com/achouhan09/mcg-osd-deployer/blob/mcg-console-status/controllers/managedmcg_controller.go#L652-L656 with `CheckReplicaStatus`.~~
- Please replace `https://github.com/achouhan09/mcg-osd-deployer/blob/mcg-console-status/controllers/managedmcg_controller.go#L638` with `s/Nooba/Noobaa/g`.

**Review by rexagod — APPROVED** [review]
/lgtm

**Pull request merged**